### ISSUE TITLE
[Bug]: Excessive logging of mob spawns

### ISSUE THREAD

**Issue by jpenilla** [Xalcon/TorchMaster#244]
### Torchmaster Version

21.1.3-beta

### Mod Loader

Neoforge

### Mod Loader Version

21.1.69

### What happened?

Torchmaster logs information about every mob that spawns to `debug.log`:
```
[19Oct2024 17:28:37.474] [Server thread/DEBUG] [Torchmaster/]: CheckSpawn - Reason: NATURAL, Type: minecraft:creeper, Pos: -1030.5/-17.0/775.5
[19Oct2024 17:28:37.474] [Server thread/DEBUG] [Torchmaster/]: Allowed spawn of minecraft:creeper
```
After leaving the game running for a while this can cause the file to become 10s or even 100s of megabytes large much faster than normal, which leads to unnecessary wear on SSDs.

In `torchmaster.toml` I have `logSpawnChecks = false` but it seems the option is ignored.

**Comment by Xalcon** [comment]
I agree that it is quite spammy, but normal users shouldn't see these during normal gameplay, since they are logged on the debug log level. Unfortunately, it seems that the debug log is enabled by default in forge and they don't provide users direct access to the log4j2.xml (its part of the fmlloader.jar).

I dont like this, but oh well, so be it. I'll fix it.

**Comment by Xalcon** [comment]
Thank you for reporting the issue.
The issue will be fixed in version 21.1.5 for both Neoforge and Fabric.
The version will be available, as usual, on Github Releases, Curseforge and Modrinth.
